dolfyn/adv/tools.py

```diff
@@ -0,0 +1,34 @@
+import dolfyn.adv.turbulence as turb
+import numpy as np
+
+
+def t_set(advr, n_bin, n_fft=None):
+    # note, can get more accurate t_range by reducing the bin size
```

Comment:
Can you make a docstring for this function? Take a look at the [numpydoc](https://github.com/numpy/numpy/blob/master/doc/HOWTO_DOCUMENT.rst.txt) specification. The most important sections are short-summary/'summary', 'parameters', and 'returns'.

Also, you could look at one of my docstrings. [Here](https://github.com/lkilcher/dolfyn/blob/master/dolfyn/adv/rotate.py#L12) is one that is pretty complete and simple.